app/models/manageiq/providers/openstack/infra_manager/host/operations.rb

```diff
@@ -0,0 +1,24 @@
+module ManageIQ::Providers::Openstack::InfraManager::Host::Operations
+  def self.included(base)
+    base.send :include, InstanceMethods
+    base.extend ClassMethods
+  end
+
+  module InstanceMethods
+    def ironic_fog_node
```

Comment:
Yeah, I'm not seeing any references to that mixin in the openstack cloud or infra providers.